flycheck.el

```diff
@@ -1610,6 +1610,26 @@ are mandatory.
 
      This property is optional.
 
+`:enabled FUNCTION'
+     A function to determine whether to use the syntax checker in
+     the current buffer.
+
+     This function behaves as `:predicate', except that it's only
+     called the first time a syntax checker is to be used in a buffer.
+
+     If the function returns a non-nil value the checker is put in a
+     whitelist to prevent further invocations of `:enabled', and if it
```

Comment:
Mention the variable name, i.e. "…is whitelisted in \`flycheck-enabled-checkers' to prevent…".  Also, "and if…" makes for a long sentence.  I'd start a new one, i.e "Otherwise it is disabled via \`flycheck-disabled-checkers' to prevent…"